get_ecg.py

```diff
@@ -0,0 +1,173 @@
+import numpy as np
+import math
+
+
+class GetEcg:
+
+    """
+    Creating this class to convert raw ECG data of all types into HR data
+    and clinical indication of brady-/tachycardia.
+    """
+    min2sec = 60
```

Comment:
constants should be all caps in snake_case: https://www.python.org/dev/peps/pep-0008/#constants